config/default.yml

```
@@ -61,6 +61,10 @@ Lint/NumberConversion:
     - fortnights
     - in_milliseconds
 
+Rails:
+  DocumentationBaseURL: https://docs.rubocop.org/rubocop-rails
+  Enabled: true
```

Can you tweak it? `Enabled` is probably a more important parameter than `DocumentationBaseURL`.

```suggestion
  Enabled: true
  DocumentationBaseURL: https://docs.rubocop.org/rubocop-rails
```